pom.xml

```diff
@@ -37,6 +120,15 @@
     </dependencies>
     
     <properties>
+        <netty-tcnative-version>2.0.46.Final</netty-tcnative-version>
+        <!-- We normally get this from common, but Vertx is built against this -->
+        <!-- Note: `netty` depends on `tcnative` and if we bump `netty`
+             we might need to bump `tcnative`, too.
+             Please check top level `pom.xml` at https://github.com/netty/netty
+             for the netty version we bump to (ie, corresponding git tag),
+             to find the correct `tcnative` version. -->
+        <netty.version>4.1.84.Final</netty.version>
+        <netty-codec-http2-version>4.1.84.Final</netty-codec-http2-version>
```

Comment:
There are different versions in ksql master and ksql 7.1.x. Aren't these versions here (in master) going to cause problems with ksql master?

For instance:
ksql master has:
```
        <netty-tcnative-version>2.0.54.Final</netty-tcnative-version>
        <netty.version>4.1.78.Final</netty.version>
        <netty-codec-http2-version>4.1.78.Final</netty-codec-http2-version>
```
ksql 7.1.x has
```
        <netty-tcnative-version>2.0.54.Final</netty-tcnative-version>
        <netty.version>4.1.84.Final</netty.version>
        <netty-codec-http2-version>4.1.84.Final</netty-codec-http2-version>
```
ksql 6.1.x
```
        <netty-tcnative-version>2.0.46.Final</netty-tcnative-version>
        <netty.version>4.1.84.Final</netty.version>
        <netty-codec-http2-version>4.1.84.Final</netty-codec-http2-version>
```

Should we change each ksql-images branch to the same version in ksql?

Reply:
Yes, the netty versions in the `ksql` repository are a mess. Basically, the problem can also be resolved by just setting all versions in `ksql` to `4.1.78.Final` and `2.0.46.Final`, because I think in https://confluentinc.atlassian.net/browse/KSE-1337 (@cprasad1) we basically just upgraded netty versions up to ksql 7.1.x further than 7.2.x and onward, which is a bit weird. But not sure if this is a good idea in the very short term because we really want to unblock these docker images and all the testing has already happened with the current versions in the `ksql` repository. 

So the idea in this PR was just to replicate the exact versions that we have in `ksql` inside `ksql-images` - because we do almost all our testing with the `ksql` versions, and then we deploy docker images with a different version, so this is risky.